TST: TestDispatcherMethods.test_inspect_cfg fails

Just installed numba package from numba channel:
```
llvmlite                  0.18.0.dev              py35_10    numba
numba                     0.33.0.dev         np111py35_33    numba
```
Running `python -m numba.tests.test_dispatcher TestDispatcherMethods.test_inspect_cfg` results in:
```
/localdisk/work/amalakho/miniconda3/envs/_test/lib/python3.5/site-packages/numba/tests/test_dispatcher.py:526: DeprecationWarning: Please use assertRegex instead.
  self.assertRegexpMatches(str(cfg), prefix)
F
======================================================================
FAIL: test_inspect_cfg (__main__.TestDispatcherMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/localdisk/work/amalakho/miniconda3/envs/_test/lib/python3.5/site-packages/numba/tests/test_dispatcher.py", line 532, in test_inspect_cfg
    check_display(cfg)
  File "/localdisk/work/amalakho/miniconda3/envs/_test/lib/python3.5/site-packages/numba/tests/test_dispatcher.py", line 526, in check_display
    self.assertRegexpMatches(str(cfg), prefix)
AssertionError: Regex didn't match: '^digraph "CFG for \\\'_ZN5numba' not found in 'digraph "CFG for \'_ZN8__main__21TestDispatcherMethods16test_inspect_cfg12$3clocals$3e7foo$244E5ArrayIdLi2E1C7mutable7alignedE\' function" {\n\tlabel="CFG for \'_ZN8__main__21TestDispatcherMethods16test_inspect_cfg12$3clocals$3e7foo$244E5ArrayIdLi2E1C7mutable7alignedE\' function";\n\n\tNode0x22de800 [shape=record,label="{entry:\\l  tail call void @NRT_incref(i8* %arg.the_array.0)\\l  tail call void @NRT_incref(i8* %arg.the_array.0), !noalias !0\\l  %.90.i = icmp eq i64 %arg.the_array.2, 0\\l  br i1 %.90.i, label %B0.endif, label %B22.endif.i.preheader, !prof !3\\l|{<s0>T|<s1>F}}"];\n\tNode0x22de800:s0 -> Node0x22dfc30[label="W:1"];\n\tNode0x22de800:s1 -> Node0x227b630[label="W:99"];\n\tNode0x227b630 [shape=record,label="{B22.endif.i.preheader:                            \\l  br label %B22.endif.i\\l}"];\n\tNode0x227b630 -> Node0x1f38960;\n\tNode0x1f38960 [shape=record,label="{B22.endif.i:                                      \\l  %c.08.i = phi double [ %.288.i, %B22.endif.i ], [ 0.000000e+00,\\l... %B22.endif.i.preheader ]\\l  %.92.07.i = phi i64 [ %.198.i, %B22.endif.i ], [ 0, %B22.endif.i.preheader ]\\l  %scevgep = getelementptr double, double* %arg.the_array.4, i64 %.92.07.i\\l  %.198.i = add nuw nsw i64 %.92.07.i, 1\\l  %.279.i = load double, double* %scevgep, align 8, !noalias !0\\l  %.288.i = fadd double %c.08.i, %.279.i\\l  %.199.i = icmp slt i64 %.198.i, %arg.the_array.2\\l  br i1 %.199.i, label %B22.endif.i, label %B0.endif, !prof !4\\l|{<s0>T|<s1>F}}"];\n\tNode0x1f38960:s0 -> Node0x1f38960[label="W:99"];\n\tNode0x1f38960:s1 -> Node0x22dfc30[label="W:1"];\n\tNode0x22dfc30 [shape=record,label="{B0.endif:                                         \\l  %c.0.lcssa.i = phi double [ 0.000000e+00, %entry ], [ %.288.i, %B22.endif.i ]\\l  tail call void @NRT_decref(i8* %arg.the_array.0), !noalias !0\\l  tail call void @NRT_decref(i8* %arg.the_array.0)\\l  store double %c.0.lcssa.i, double* %retptr, align 8\\l  ret i32 0\\l}"];\n}\n'

----------------------------------------------------------------------
Ran 1 test in 0.465s

FAILED (failures=1)
```

Please confirm if you see this issue.

--- a/numba/tests/test_dispatcher.py
+++ b/numba/tests/test_dispatcher.py
@@ -522,7 +522,9 @@ class TestDispatcherMethods(TestCase):
             # simple stringify test
             if wrapper:
                 wrapper = "{}{}".format(len(wrapper), wrapper)
-            prefix = r'^digraph "CFG for \'_ZN{}5numba'.format(wrapper)
+            module_name = __name__.split('.', 1)[0]
+            module_len = len(module_name)
+            prefix = r'^digraph "CFG for \'_ZN{}{}{}'.format(wrapper, module_len, module_name)
             self.assertRegexpMatches(str(cfg), prefix)
             # .display() requires an optional dependency on `graphviz`.
             # just test for the attribute without running it.

